utils/helpers.js

```diff
@@ -4,7 +4,7 @@ export const getClass = (styles, className) => className && classnames(className
 
 export const breakOnSpaceChars = (str) => str.split('\\n');
 
-export const capitalizeFirstLetter = (string) => string.charAt(0).toUpperCase() + string.slice(1);
+export const capitalizeFirstLetter = (string) => string ? string.charAt(0).toUpperCase() + string.slice(1) : string;
```

Comment:
Евентуално може една идея по-секси с:

```suggestion
export const capitalizeFirstLetter = (string = '') => string.charAt(0).toUpperCase() + string.slice(1);
```

Но имам по-генерален коментар - дали там, където се ползва capitalizeFirstLetter и се е стигнало до краша, не трябва да се разследва защо е имало undefined/празен текст?